.chloggen/msg_cleanup-scraper-helper.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: scraperhelper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Removing deprecated parameter from `NewDefaultScraperControllerSettings`
```

Comment:
The function itself has not been deprecated, so I don't think we should do it until we have properly deprecated things